CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 - Sortition processing performance has been improved by about an order of
-  magnitude, by avoiding a slew of expensive database reads (#3048).
+  magnitude, by avoiding a slew of expensive database reads (#3045).
```

Comment:
This should mention (and warn) about the schema migration. Restarting a node, using the an existing chainstate directory, with this change will require more time than other updates have in the past, so users should get some kind of warning in the changelog (either in this entry, or at the top level content for the release).

Reply:
Fixed